erpnext/manufacturing/doctype/production_plan/production_plan.py

```diff
@@ -62,11 +62,10 @@ def get_pending_material_requests(self):
 			mr_filter += " and mr.transaction_date <= %(to_date)s"
 		if self.warehouse:
 			mr_filter += " and mr_item.warehouse = %(warehouse)s"
-
 		if self.item_code:
-			item_filter += " and mr_item.item_code = %(item)s"
+			item_filter += " and mr_item.item_code like %(item)s"
 
-		pending_mr = frappe.db.sql("""
+		pending_mr = frappe.db.sql("""   # nosemgrep
```

Comment:
```suggestion
		pending_mr = frappe.db.sql(""" 
```

This is becoming part of the query string. Remove it, I will be fixing that sql rule.